src/models/rapTb.js

```diff
@@ -0,0 +1,13 @@
+export default {
+  fields: {
+      video_link :     "text",
+      lyrics     :     "text",
+      label      :     "text",
+      date_of_release :"date",
```

Comment:
```suggestion
      date_of_release :{
      type:"date",
      default: {"$db_function": "toTimestamp(now())"}
      },
```
It is better to have a good timestamp default function

Reply:
What does the file name `rapTb.js` mean?

Reply:
@yashgangwar7558  @AmitKarnam  @AbhijithGanesh would you like to see any changes in the file naming?